src/api/goods.ts

```diff
@@ -1,11 +1,24 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
-}
+export const getAll = async (): Promise<Good[]> => {
+  try {
+    const response = await fetch(API_URL);
 
-export const get5First = () => {};
+    return await response.json();
+  } catch (error: any) {
+    throw new Error(`${error.status} - ${error.statusText}`);
+  }
+};
 
-export const getRedGoods = () => {};
+export const get5First = async () => {
+  const goods = await getAll();
+
+  return goods.slice(0, 5);
+};
```

Comment:
Actually it not only first 5 its 
Load 5 first goods shows 5 first goods after sorting them by name

Reply:
I missed that, thank you!